csharp-advanced-topics/ActionAndFuncInCsharp/ActionAndFunc/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ActionAndFunc
+{
+    public class Program
+    {
+        private const string Quote = "I am the one who knocks.";
```

Comment:
Please introduce some line breaks between fields, methods, and logical units in the methods. The code feels very stuffed.

Reply:
Sorry, i couldn't understand. Do you want this ? 
```
public class Program {

        private const string Quote = "I am the one who knocks.";
```

Reply:
Hey Isa, you should introduce newlines here:
![image](https://user-images.githubusercontent.com/29179238/149663076-4c831355-879e-4f49-920b-1ab12479c9bd.png)
And also here:
![image](https://user-images.githubusercontent.com/29179238/149663106-afc52240-aaf9-4b9d-b63f-4055e3f363b7.png)
Applies to the test class as well.